### PR TITLE
Vboulos obs change imagename

### PR DESCRIPTION
**Pull request by vboulos**
- Modify the Obs image by replacing underscore "_" with dash "-"

### CODE REVIEW

File: ci-operator/jobs/stolostron/application-ui-test/OWNERS
Comment (ascerra) [inline ci-operator/jobs/stolostron/application-ui-test/OWNERS:11]: If we remove these now they will just be adjusted automatically by the bot do we even want to bother removing these OWNERS?

Comment (vboulos) [reply ci-operator/jobs/stolostron/application-ui-test/OWNERS:11]: Do we need to add them back in the PR? Or the bot will handle it?
If the bot will do it then fine let it do it :)

Comment (ascerra) [reply ci-operator/jobs/stolostron/application-ui-test/OWNERS:11]: You can just update this PR to not remove them as a part of this PR

Comment (vboulos) [reply ci-operator/jobs/stolostron/application-ui-test/OWNERS:11]: cool, let the bot do it :)

**Review by ascerra — APPROVED** [review]

File: ci-operator/config/stolostron/observability_core_automation/OWNERS
Comment (ascerra) [inline ci-operator/config/stolostron/observability_core_automation/OWNERS:15]: I believe we need to just add a new line at the end of the file to match the one that already exists to resolve this discrepancy.

**Review by ascerra — APPROVED** [review]

**Comment by evelinec** [comment]
@jianlinliu @liangxia can you help to review the PR and let us know if there's any feedback before we can merge? This PR is needed for the ACM team to test one of our last components as part of the product tests on OCP CI. Thanks.

**Comment by evelinec** [comment]
@vboulos since this PR has one review approval from @ascerra, is it ready to merge so you can start the Obs testing? The other reviewers, which looks like are automatically/randomly selected by the openshift-ci bot are not part of this project.

**Comment by vboulos** [comment]
@evelinec yes the Openshift-ci bot is assigning people that are not part of this project, so we are going to figure it out and fix it.
So yes when it gets merged I can start testing Obs.

**Comment by evelinec** [comment]
@vboulos Not sure what need to be figured out. If we're looking for OCP CI folks to review any PRs, @jianlinliu @liangxia are the correct contacts.

**Review by chenz4027 — APPROVED** [review]
/lgtm

**Review by ascerra — APPROVED** [review]

**Comment by ascerra** [comment]
/pj-rehearse [pull-ci-stolostron-observability_core_automation-release-2.7-images](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/stolostron_observability_core_automation/108/pull-ci-stolostron-observability_core_automation-release-2.7-images/1643403028218253312)

**Comment by ascerra** [comment]
/pj-rehearse pull-ci-stolostron-observability_core_automation-release-2.7-images

**Pull request merged**
